docs/installation-contributors.md

```diff
@@ -12,12 +17,10 @@ These steps are required once per machine.
 The server needs to access some metadata about the build configuration. This
 data are produced by an sbt plugin.
 
-You can install the plugin with (replace `<version>` with the latest available version)
-
-[![Download](https://api.bintray.com/packages/scalameta/sbt-plugins/sbt-metals/images/download.svg)](https://bintray.com/scalameta/sbt-plugins/sbt-metals/_latestVersion)
+You can install the plugin with:
 
 ```scala
-addSbtPlugin("org.scalameta" % "sbt-metals" % "<version>")
+addSbtPlugin("org.scalameta" % "sbt-metals" % "@VERSION@")
```

Comment:
The "N.B" and "troubleshooting tips" links on this page give a 404

```md
***N.B.***  This project is still in development - right now you will need to run the VSCode plugin as described in the [contributing guide](../CONTRIBUTING.md#running-a-local-version-of-the-vscode-extension)

...

[**Troubleshooting tips**](../CONTRIBUTING.md#troubleshooting)
```

Seems we can remove them.